# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../isdn4k-utils/strange-func-decl.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- isdn4k-utils/isdnlog/isdnlog/processor.c~	2002-04-01 20:06:10.000000000 +0200
+++ isdn4k-utils/isdnlog/isdnlog/processor.c	2004-10-14 14:23:54.634300688 +0200
@@ -5293,7 +5293,7 @@
     tei = BROADCAST; /* Wenn nach einer tei-Zeile keine hex:-Zeile kommt, tei ungueltig machen! */
 
     if ((type == SETUP) && !replay) { /* fetch additional info from "/dev/isdninfo" */
-      static void moreinfo(); /* soviel zu Objektorientiertem Denken ;-) */
+      /*static void moreinfo(); ??? soviel zu Objektorientiertem Denken ;-) */
       moreinfo();
     } /* if */
 
